### PR TITLE
Update install/linux/docker-ce/debian.md

### PR DESCRIPTION
**Pull request by erikmd**
### Proposed changes

This PR proposes to update the page https://docs.docker.com/install/linux/docker-ce/debian/ :

* Keep the name of the latest stable distribution up-to-date;
* Mention in the Debian install procedure, the same APT packages as in [Ubuntu's install procedure](https://docs.docker.com/install/linux/docker-ce/ubuntu/) (to be more precise, `gnupg2` was just a transitional package providing symlinks to `gnupg` and depending on `gnupg-agent`, and IINM `gnupg-agent` should be enough for Debian as well :)  
    see also [this commit message](https://github.com/docker/docker.github.io/commit/651ae2d507fac4fa5cad983cdbc0288158164965) for details.

### CODE REVIEW

**Comment by erikmd** [comment]
friendly ping :)

**Comment by usha-mandya** [comment]
Hey @erikmd, apologies for the delay. We'll review this update asap and get back to you. Thanks for updating the docs.

File: install/linux/docker-ce/debian.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ from the repository.
         apt-transport-https \
         ca-certificates \
         curl \
-        gnupg2 \
+        gnupg-agent \
```
Comment (thaJeztah) [inline install/linux/docker-ce/debian.md]: 👍 tested that the package exists both for debian stretch and debian buster

**Comment by erikmd** [comment]
Thanks @usha-mandya @thaJeztah :+1:

**Pull request merged**